test/db/cmd/metadata

```diff
@@ -21,7 +21,7 @@ CMDS=<<EOF
 af
 CCf-*~?
 CC hello
-CC world @ $$ + 32
+CC world @ $$+32
```

Comment:
do this mean that spaces are no longer supported in math expressions?

Reply:
It depends... `cmd @ arg` expects just one argument after @ , so if you put a space it becomes 3 arguments... We agreed on not having the concept of "math expressions" in the parser. I may try to fix this, but I'm not 100% sure it will be easy